app/models/validations/tenancy_validations.rb

```diff
@@ -32,4 +32,10 @@ def validate_fixed_term_tenancy(record)
   def validate_other_tenancy_type(record)
     validate_other_field(record, 3, :tenancy, :tenancyother)
   end
+
+  def validate_joint_tenancy(record)
+    if record.hhmemb == 1 && record.joint != 2 && record.collection_start_year == 2022
+      record.errors.add :joint, I18n.t("validations.tenancy.not_joint")
```

Comment:
Need to add this to `hhmemb` too since you could trigger it from either page

Reply:
Done

Reply:
@baarkerlounger @dushan-madetech The error message looks fine to me, as we reference 'household', so should make sense if triggered by either question

Reply:
@Dan13L3 would you not want it reversed so that if it's the household members you're filling out it reads:

"There must be more than 1 person in the household as you told us this is a joint tenancy"

Reply:
@baarkerlounger Ahh, I misunderstood - I thought we were saying we could only have a single error for both questions. But yes, agree with your suggestion. Although we'd need to tweak slightly to: "There must be more than one person in the household as you told us this is a joint tenancy"

Reply:
@Dan13L3 @baarkerlounger Updated the message - check the screenshots I've attached to the PR

Reply:
Thanks @dushan-madetech - looks good to me